### PR TITLE
Link Apis to actual backend

### PR DESCRIPTION
**Pull request by HarryPi**
### Code changes
- Change the API path to the static API path

### CODE REVIEW

File: AdminClient/src/environments/enviroment.common.ts [hunk 1]
```diff
@@ -5,5 +5,5 @@ export const commonEnvironment = {
   CANCELLATION_TEXT: 'Cancel',
   DELETE_CUSTOMER_CONFIRM_TEXT: 'Are you sure you want to delete this customer?',
   DELETE_CUSTOMER_TITLE: 'Delete Customer',
-  apiRoot: () => 'http://localhost:8080/api'
+  apiRoot: () => 'http://40.65.236.154/api'
```
Comment (markscamilleri) [inline AdminClient/src/environments/enviroment.common.ts:8]: Should we leave this to be set in a config by the deployment pipeline? (Also useful for helm)

File: Backend/src/main/java/com/project/omega/bean/dto/ProductDTO.java [hunk 2]
```diff
@@ -3,7 +3,6 @@
 import com.project.omega.bean.dao.entity.Supplier;
 
 public class ProductDTO {
-    private Long id;
```
Comment (markscamilleri) [inline Backend/src/main/java/com/project/omega/bean/dto/ProductDTO.java:6]: We're removing ids?

File: Backend/src/main/java/com/project/omega/controller/OrderController.java [hunk 3]
```diff
@@ -135,7 +135,7 @@ private void validateProductsExistence(List<OrderProductDto> orderProducts) thro
                 list.add(op);
             }
         }
-        if (!CollectionUtils.isEmpty(list)) {
+        if (CollectionUtils.isEmpty(list)) {
             new ProductNotFoundException("Product not found");
```
Comment (markscamilleri) [inline Backend/src/main/java/com/project/omega/controller/OrderController.java:139]: Would this return a status 200 with an empty list?

File: RepClient/src/app/home/home.page.html [hunk 4]
```diff
@@ -4,18 +4,32 @@
...
+						<p>We couldnt find any suppliers registered with you yet pelase contact your administrator
+						to add them!</p>
+					</ion-card-content>
+					<hr>
+				</ion-card>
+			</ion-col>
 			<ion-col size="6" *ngFor="let company of companies">
 				<ion-card>
-					<img [src]="company.imageUrl" alt=""/>
+					<!--<img [src]="company.imageUrl" alt=""/>-->
```
Comment (markscamilleri) [inline RepClient/src/app/home/home.page.html:23]: Should we provide a dummy image?

File: AdminClient/src/environments/enviroment.common.ts (hunk 1 above)
Comment (HarryPi) [reply AdminClient/src/environments/enviroment.common.ts:8]: yes that would be a good idea but that would not work for development we can have in the .prod env the enviromental variable and on the .prod this

File: Backend/src/main/java/com/project/omega/bean/dto/ProductDTO.java (hunk 2 above)
Comment (HarryPi) [reply Backend/src/main/java/com/project/omega/bean/dto/ProductDTO.java:6]: Just from the DTO as we dont have control over it no need to change any models as any property that java doesnt find at the api end point will simply ignore

File: Backend/src/main/java/com/project/omega/controller/OrderController.java (hunk 3 above)
Comment (HarryPi) [reply Backend/src/main/java/com/project/omega/controller/OrderController.java:139]: This is a function that is called in a put request, so it makes sense if any of the product ids fails to match to be a bad request.

It simply ensures that all bound products actually exist on the DB.

File: RepClient/src/app/home/home.page.html (hunk 4 above)
Comment (HarryPi) [reply RepClient/src/app/home/home.page.html:23]: I dont think so coz that would invite the question why didnt u put actual images ;p

**Pull request merged**